src/main/scala/io/github/paoloboni/http/HttpClient.scala

```diff
@@ -100,8 +97,8 @@ sealed class HttpClient[F[_]: Async: Client: LogWriter](implicit
         }
       }
       handled <- decoded.fold(
-        decodingFailure => E.raiseError(decodingFailure),
-        response => F.pure(response)
+        decodingFailure => implicitly[Async[F]].raiseError(decodingFailure),
+        response => response.pure[F]
```

Comment:
I like this syntax!

Reply:
yeah i like the syntax of calling `.pure[F]` much more than calling `F.pure(...)`, this obviously works for all kinds of effects which is  quite handy